src/DefaultHandle.jsx

```diff
@@ -65,7 +61,7 @@ export default withStyles(({ color, unit }) => ({
     zIndex: 2,
     boxShadow: `0 ${unit / 4}px ${unit / 4}px ${color.textDisabled}`,
     ':focus': {
-      boxShadow: `${color.focus} 0 0 2px 2px`,
+      boxShadow: `${color.teal} 0 0 1px 1px`,
```

Comment:
I think we might want to keep this as `color.focus` and just have it map to teal in the theme so that the variable is meaningful as to what it represents.